.github/workflows/test.yml

```diff
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main, 717-add-dbtrunner-local-executor-2] # # TODO:remove before merge
```

Comment:
Commenting here to remind me to remove this before merging. The performance dag was updated to use postgres and wanted to test the workflow in this PR.